src/fairchem/core/modules/element_references.py

```diff
@@ -0,0 +1,206 @@
+"""
+Copyright (c) Meta, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+from __future__ import annotations
+
+import logging
+from functools import partial
+from typing import TYPE_CHECKING
+
+import numpy as np
+import torch
+from torch import nn
+from torch.utils.data import DataLoader, Dataset
+from tqdm import tqdm
+
+from fairchem.core.datasets import data_list_collater
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from torch_geometric.data import Batch
+
+
+class LinearReference(nn.Module):
+    """Compute a linear reference for target scalar properties"""
```

Comment:
nit - can we describe what we mean by "linear reference" for others. I don't think this is obvious.

I wrote out a description in the OC22 paper - https://arxiv.org/pdf/2206.08917 page 34.

Reply:
Done!